.github/CONTRIBUTING.md

```diff
@@ -5,85 +5,14 @@ Open MPI!
 
 ![You're awesome!](https://www.open-mpi.org/images/youre-awesome.jpg)
 
-General information about contributing to the Open MPI project can be found at the [Contributing to Open MPI webpage](https://www.open-mpi.org/community/contribute/).
-The instructions below are specifically for opening issues and pull requests against Open MPI.
+General information about contributing to the Open MPI project can be
+found at the [Contributing to Open MPI
+webpage](https://docs.open-mpi.org/en/main/contributing.html).
 
-## Content
+In short: Open MPI is hosted on GitHub, and we gladly accept pull
+requests.  The instructions below are specifically for opening issues
```

Comment:
Did you want to keep the line "The instructions below..."?

Reply:
That's a valid point; thanks.  I'll open a new PR.